Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSource.java

```diff
@@ -152,16 +150,19 @@ private String getFaceImageUrl(Proxy proxy, CardDownloadData card, boolean isTok
             }
         } else {
             // BY CARD NUMBER
-            // localized and default
+            // localized and none
```

Comment:
It's not "none" -- it's same default image (even without loc code in url).

Reply:
Returned "none" back to "default".